### DISCUSSION TITLE
Radial / snapping previews in Settings panel

### DISCUSSION THREAD

**Discussion by greg1075** [mrkai77/Loop#470] [Q&A]
Just a general remark / suggestion

The new settings menu is great, but I find showing the right panel radial menu / window snapping previews at all times unecessary and overkill. You could simply things and make away with it for all the submenus that don't require any sort of preview, and maybe make it slide out for the those that do? 

I'd remove it for Theming>Icon, Settings>Keybindings, Loop>Advanced, Loop>Excluded Apps and Loop>About. Unless you think it makes more sense for UI consistency's sake. 

Also, why is a separate submenu for Accent Color when it could (should?) be included in Theming>Radial Menu?

Cheers.

**Comment by SenpaiHunters** [comment]
Cheers for the suggestion, we've been looking at a simpler UI to show, and maybe move things around 👍

**Reply by mrkai77** [reply]
Yes!

> Also, why is a separate submenu for Accent Color when it could (should?) be included in Theming>Radial Menu?

The accent color also does not only affect the radial menu, but affects the theming around the entire app.

**Comment by JaceThings** [comment]
> but I find showing the right panel radial menu / window snapping previews at all times unecessary and overkill. 

It can now be hidden in 1.1 :)
